packages/extension/src/store/modules/ui.js

```diff
@@ -71,12 +80,22 @@ export default {
     topSitesInstructions(state) {
       return state.instructionsStep === 0;
     },
+
     sidebarInstructions(state) {
       return state.instructionsStep === 1;
     },
+
     showReadyModal(state) {
       return state.instructionsStep === 2;
     },
+
+    dndMode(state) {
+      return state.dndModeTime !== null;
+    },
+
+    dndModeTime(state) {
```

Comment:
This getter is redundant, you can simply access the `dndModeTime` in state

Reply:
Hmmmm, I added it as a helper to avoid always calling
`dndModeTime !== null`